pkg/storage/boltdb/boltdb.go

```diff
@@ -0,0 +1,321 @@
+package boltdb
+
+import (
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"go.etcd.io/bbolt"
+	"zotregistry.io/zot/errors"
+	zlog "zotregistry.io/zot/pkg/log"
+	"zotregistry.io/zot/pkg/storage/cachedb"
+	"zotregistry.io/zot/pkg/storage/constants"
+)
+
+type Driver struct {
+	rootDir     string
+	db          *bbolt.DB
+	log         zlog.Logger
+	useRelPaths bool // weather or not to use relative paths, should be true for filesystem and false for s3
```

Comment:
"whether"

Reply:
and this should be derived automatically from the type of storage

Reply:
Removed useRelPaths from config, it gets set from CreateDatabaseDriver according to storage type